File: src/Maintenance.java
public class Maintenance {
    private Employee employee;
    private Animal animal;
    private String procedure;
    private String procedureDate;
    private String observations;

    public Maintenance(Employee employee, Animal animal, String procedure, String procedureDate) {
        this.employee = employee;
        this.animal = animal;
        this.procedure = procedure;
        this.procedureDate = procedureDate;
    }

    public Maintenance(Employee employee, Animal animal, String procedure, String procedureDate, String observations) {
        this.employee = employee;
        this.animal = animal;
        this.procedure = procedure;
        this.procedureDate = procedureDate;
        this.observations = observations;
    }
    public String consult(){
        if(observations == null){
            return "Employee: "+employee.getName()+"\nAnimal ID: "+animal.getId()+"\nProcedure: "+procedure+"\nDate: "+procedureDate;
        }else{
            return "Employee: "+employee.getName()+"\nAnimal ID: "+animal.getId()+"\nProcedure: "+procedure+"\nDate: "+procedureDate+"\nObservations: "+observations;
        }

    }
    public Employee getEmployee(){
        return employee;
    }

    public Animal getAnimal(){
        return animal;
    }

    public String getProcedure() {
        return procedure;
    }

    public String getProcedureDate() {
        return procedureDate;
    }

    public String getObservations() {
        return observations;
    }
}

File: src/Maintenance.py
from Animal import Animal
from Employee import Employee


class Maintenance:
    __employee = Employee
    __animal = Animal
    __procedure = ""
    __procedure_date = ""
    __observations = ""

    def __init__(self, employee, animal, procedure, procedure_date, observations=None):
        self.__employee = employee
        self.__animal = animal
        self.__procedure = procedure
        self.__procedure_date = procedure_date
        self.__observations = observations

    def consult(self):
        if self.__observations == "":
            return (f"Employee: {self.__employee.getName()} \nAnimal ID: {self.__animal.getId()} \nProcedure: "
                    f"{self.__procedure} \nDate: {self.__procedure_date}")
        else:
            return (f"Employee: {self.__employee.getName()} \nAnimal ID: {self.__animal.getId()} \nProcedure: "
                    f"{self.__procedure} \nDate: {self.__procedure_date} \nObservations: {self.__observations}")

    def get_employee(self):
        return self.__employee

    def get_animal(self):
        return self.__animal

    def get_procedure(self):
        return self.__procedure

    def get_procedure_date(self):
        return self.__procedure_date

    def get_observations(self):
        return self.__observations